### DISCUSSION TITLE
Reply to an inline code comment thread

### DISCUSSION THREAD

**Discussion by NathanKleekamp** [pwntester/octo.nvim#256] [Q&A] [answered]
How do I post a reply to another user's comment about a line of code? Right now, if I type `<space>ca` it adds the cursor to the text area for the original comment; it doesn't add a threaded response with a prompt. What am I missing / doing wrong?

**Comment by NathanKleekamp** [comment, marked as answer]
I figured this out on my own. Seems that if you want to reply to an inline comment, you need to start a review `:Octo review start` and then find the comment thread and run `:Octo comment add`

**Reply by mhanberg** [reply]
It seems that you then have to submit a new review for the comment to actually publish, or else it stays in the "pending" state
